model/commitqueue/dequeue_sender_test.go

```diff
@@ -46,7 +46,7 @@ func TestCommitQueueDequeueLogger(t *testing.T) {
 	assert.NoError(err)
 	assert.False(q.Processing)
 	assert.Equal("2", q.Queue[0].Issue)
-	eventLog, err := event.FindUnprocessedEvents()
+	eventLog, err := event.FindUnprocessedEvents(0)
```

Comment:
Should these be sending the default limit? I'm actually not sure what a limit of 0 means in our driver wrapper

Reply:
It definitely means something greater than 0 (empirically).
My guess would be that it means no limit, like it means for [the shell](https://docs.mongodb.com/manual/reference/method/cursor.limit/#zero-value)

I changed it to the default limit to avoid the uncertainty.